scss/docs/global-nav.scss

```diff
@@ -0,0 +1,831 @@
+// FOR DEMO PURPOSES ONLY:
```

Comment:
As this is for examples only, I'd put it as `.css` file in the `/examples/layouts/` folder and `{% include %}` it into styles (just as we do with scripts).